docs/unified-test-documentation/dasharo-compatibility/315-network-boot.md

```diff
@@ -234,3 +234,72 @@ server.
 1. The iPXE application boots successfully.
 1. iPXE obtains an IP address.
 1. iPXE boots an `Debian 11` from netboot.xyz.
+
```

Comment:
Keeping in mind that this is a new function, we don't need to update any test matrix?

Reply:
Test matrices for MSI z690 and Novacustom platforms are similar to:

```
| 14.  | [Network boot][PXE]                                    | PXE           | Without PXE007.001                   |
```

Will other platforms get support for this functionality?